index.html

```diff
@@ -15,13 +15,13 @@
 	</nav>
 	<section class="homepage-hero">
 		<!-- Hero section, the cover for the page -->
-		<div class="homepage-hero__layer"></div>
-		<article class="homepage-hero__article">
-			<div class="homepage-hero__headline">
+    <div class="homepage-hero__layer"></div>
+    <div class="homepage-hero__hdlp">
```

Comment:
The logo don't need a wrapper div to align it. You can remove it and center de logo inside the section.